spec/controllers/wishlists_controller_spec.rb

```diff
@@ -75,6 +75,31 @@
         end
       end
     end
+
+    describe "GET #show" do
+      context "when wishlist is created by logged in user" do
+        before do
+          new_wishlist = create(:wishlist, user: controller.current_user)
+          get_show(new_wishlist.id)
+        end
+
+        it "returns 200 HTTP status code" do
+          expect(response).to have_http_status(200)
+        end
+
+        it "renders the 'show' template" do
+          expect(response).to render_template(:show)
+        end
+      end
+
+      context "when wishlist is not created by logged in user" do
+        before(:each) { get_show(id: 0) }
```

Comment:
Doesn't match the message from `context`. Create the wishlist with another user, and actually try to access it.